test/expand.ts

```diff
@@ -14,7 +13,7 @@ import {
 const operandToAbstractSQL = operandToAbstractSQLFactory();
 import test from './test.js';
 
-const createAggregate = function (args) {
+const createAggregate = function (args: any) {
```

Comment:
I'm not sure on why this change was made when nothing seems to have prompted it to be necessary? And the same for a bunch of other places that add an `any` without any corresponding change that would appear to make it now necessary